src/eredis_cluster.appup.src

```diff
@@ -27,7 +27,7 @@
       {load_module, eredis_cluster_pool_worker, brutal_purge, soft_purge, []},
       {load_module, eredis_cluster_monitor, brutal_purge, soft_purge, []}
     ]},
-    {<<"0.6.[34]">>, [
+    {<<"0.6.[345]">>, [
       {load_module, eredis_cluster_pool_worker, brutal_purge, soft_purge, []}
     ]},
     {<<".*">>, [
```

Comment:
Create another clause for upgrading from 0.6.7 -> 0.6.8, like this:

```
{"0.6.8",
  [
    {"0.5.11", [
      {load_module, eredis_cluster_sup, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_pool_worker, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_monitor, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_pool, brutal_purge, soft_purge, []}
    ]},
    {"0.6.2", [
      {load_module, eredis_cluster_sup, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_pool_worker, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_monitor, brutal_purge, soft_purge, []}
    ]},
    {<<"0.6.[3-6]">>, [
      {load_module, eredis_cluster_sup, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_pool_worker, brutal_purge, soft_purge, []}
    ]},
    {"0.6.7", [
      {load_module, eredis_cluster_sup, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster, brutal_purge, soft_purge, []}
    ]},
    {<<".*">>, [
    ]}
  ],
  [
    {"0.5.11", [
      {load_module, eredis_cluster_sup, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_pool_worker, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_monitor, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_pool, brutal_purge, soft_purge, []}
    ]},
    {"0.6.2", [
      {load_module, eredis_cluster_sup, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_pool_worker, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_monitor, brutal_purge, soft_purge, []}
    ]},
    {<<"0.6.[3-6]">>, [
      {load_module, eredis_cluster_sup, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster_pool_worker, brutal_purge, soft_purge, []}
    ]},
    {"0.6.7", [
      {load_module, eredis_cluster_sup, brutal_purge, soft_purge, []},
      {load_module, eredis_cluster, brutal_purge, soft_purge, []}
    ]},
    {<<".*">>, [
    ]}
  ]
}.
```

Note that I also added the 2 load_module instructions into older versions, because upgrade from old versions to the current version (0.6.8) also need to reload these 2 modules.